pleskdistup/actions/spamassassin.py

```diff
@@ -36,25 +36,31 @@ def _revert_action(self) -> action.ActionResult:
 
 
 class HandleUpdatedSpamassassinConfig(action.ActiveAction):
+    spamassasin_service_name: str
+
     # Make sure the trick is preformed before any call of 'systemctl daemon-reload'
     # because we change spamassassin.service configuration in scope of this action.
     def __init__(self) -> None:
         self.name = "handle spamassassin configuration update"
+        self.spamassasin_service_name = "spamassassin.service"
```

Comment:
The service has been renamed to `spamd` in SpamAssassin 4, available e.g. on Debian 12, so to be future-proof it's better to make the name at least configurable.